person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
```

Comment:
A method for adding rentals should be present.: [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). In order for a person to have many rentals 
 **person has many rentals**

Example: 
```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```